Formula/macvim.rb

```diff
@@ -14,20 +14,29 @@ class Macvim < Formula
   end
 
   option "with-override-system-vim", "Override system vim"
+  option "with-gettext", "Build MacVim with +gettext."
 
   deprecated_option "override-system-vim" => "with-override-system-vim"
 
+  conflicts_with "vim",
+    :because => "They both override system vim" if build.with? "override-system-vim"
```

Comment:
Huh?
[override-system-vi](https://github.com/Homebrew/homebrew-core/blob/master/Formula/vim.rb#L115) only generates `symlink "vim" => "vi"`, system global symlink `vim` is always generated whatever override-system-vi or not.